tekton/utils/pipeline.go

```diff
@@ -85,6 +86,23 @@ func (pr *PipelineRef) ToTektonPipelineRef() *tektonv1.PipelineRef {
 	return tektonPipelineRef
 }
 
+// ToTektonParams converts ParameterizedPipeline []Param to []tektonv1.Param and returns it.
+func (prp *ParameterizedPipeline) ToTektonParams() []tektonv1.Param {
```

Comment:
Not what I had in mind. I expected an array of Params to be passed. If you go this route, call it `GetTektonParams` as you are not converting a `ParameterizedPipeline` here.

Reply:
Ok. For now I renamed. We can sync later on show this should be properly done, so it goes with the refactor.